From 4bcc9e1dc8873af5e0b52efd230a903ffc80ffcb Mon Sep 17 00:00:00 2001
From: Jaehoon You <jaehoon.you@samsung.com>
Date: Mon, 9 Sep 2019 13:56:12 +0900
Subject: [PATCH] st_iot_core: add freertos header path

[Model] STDK
[BinType]
[Customer]

[Issue#]
[Request]
[Occurrence Version]

[Problem] os/freertos/iot_os_util_freertos.c:19:22: fatal error: FreeRTOS.h: No such file or directory
[Cause & Measure] freertos header path was omitted in esp32.
[Checking Method]

[Team]
[Developer] Jaehoon You
[Solution company] Samsung
[Change Type] N/A

Change-Id: Ic4304f87ffb887bd1849c599d3f6fee79cba795e
Signed-off-by: Jaehoon You <jaehoon.you@samsung.com>
---
 components/freertos/component.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/freertos/component.mk b/components/freertos/component.mk
index dae672e..e3792e7 100644
--- a/components/freertos/component.mk
+++ b/components/freertos/component.mk
@@ -3,7 +3,7 @@
 #
 
 COMPONENT_ADD_LDFLAGS += -Wl,--undefined=uxTopUsedPriority
-COMPONENT_ADD_INCLUDEDIRS := include
+COMPONENT_ADD_INCLUDEDIRS := include include/freertos
 COMPONENT_PRIV_INCLUDEDIRS := include/freertos
 
 tasks.o event_groups.o timers.o queue.o: CFLAGS += -D_ESP_FREERTOS_INTERNAL
-- 
2.7.4

